opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/ServerTracingFilter.java

```diff
@@ -59,7 +57,8 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
                 spanBuilder.asChildOf(extractedSpanContext);
             }
 
-            Span span = spanBuilder.start();
+            Span span = spanBuilder.startManual();
+            tracer.makeActive(span);
```

Comment:
@pavolloffay Would be worth creating an issue on opentracing-java to see if a general solution can be found.